app/src/main/java/org/httpsknighthacks/knighthacksandroid/Profile.java

```diff
@@ -75,6 +91,64 @@ public void onClick(View view) {
                     pickImageFromGallery();
                 }
             }
+        });*/
+
+        // TODO: The comments below are buttons that help change the activity
+        /*mNotificationSettings = findViewById(R.id.profile_settings_button);
+        changeActivityBtn(mNotificationSettings, SettingsActivity.class);
+
+        mGroupsActivity = findViewById(R.id.my_groups_btn);
+        changeActivityBtn(mGroupsActivity, Groups.class);*/
+    }
+
+    // TODO: Set mDialog to the qr code popup and set the image with user qr
+    private void showQRCodePopup() {
+        ImageView imageView = findViewById(R.id.QRCloseBtn);
+
+        try {
+            mDialog.getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
+        }
+        catch (NullPointerException e){
+
+        }
+
+        mDialog.show();
+
+        imageView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                mDialog.dismiss();
+            }
+        });
+    }
+
+    // TODO: Set mDialog to the confirm logout popup
+    private void confirmLogoutPopup() {
+        Button button = findViewById(R.id.confirmLogoutBtn);
+
+        try {
+            mDialog.getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
+        }
+        catch (NullPointerException e){
+
+        }
+
+        mDialog.show();
+
+        button.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                mDialog.dismiss();
+            }
+        });
+    }
+
+    private void changeActivityBtn(CardView mCardView, Class activityClass) {
```

Comment:
Just wondering what is the purpose of this method?

Reply:
A function to change the activity to the settings or groups; didn't want repeated code.